src/Processors/Sources/ShellCommandSource.cpp

```diff
@@ -275,7 +275,7 @@ class TimeoutWriteBufferFromFileDescriptor : public BufferWithOwnMemory<WriteBuf
             ssize_t res = ::write(fd, working_buffer.begin() + bytes_written, offset() - bytes_written);
 
             if ((-1 == res || 0 == res) && errno != EINTR)
-                throw ErrnoException(ErrorCodes::CANNOT_WRITE_TO_FILE_DESCRIPTOR, "Cannot write into pipe");
+                throw ErrnoException(ErrorCodes::SHELL_COMMAND_CANNOT_WRITE_TO_FILE_DESCRIPTOR, "Shell command cannot write into pipe");
```

Comment:
While this class is isolated inside the cpp file, it has an ambiguous name, 
which contradicts the specificity of the error code.

This is ok, but let's put it into an anonymous namespace.